src/Subpages/Resources/BackgroundReportHelp.jsx

```diff
@@ -0,0 +1,57 @@
+import { Typography, List, ListItem, } from "@material-ui/core";
+
+const BackgroundReportHelp = () => {
+  return (
+    <div style={{padding: "24px"}}>
+      <Typography style={{paddingBottom: "36px"}} variant="h3">Criminal Background Report</Typography>
+      <Typography variant="body1">
+        You might need a copy of your Criminal History Record Information (CHRI)
+        to vacate your record. There are two options:
```

Comment:
This might be more of a design/content question, but we say there are 2 options, but it's hard to tell what the two options are. Maybe we could have a bullet point list and say option 1 OR option 2